app/tsconfig.json

```diff
@@ -14,5 +14,6 @@
     "outDir": "electron/",
     "noEmit": true
   },
-  "include": ["electron/**/*"]
+  "include": ["electron/**/*", "e2e-tests/**/*"],
+  "exclude": ["node_modules"]
```

Comment:
Can you please move this to a separate commit, with an explanation of why it's necessary?